File: sanghyun/programmers-2/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String[] arr = s.split(" ");
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        for (int i = 0; i < arr.length; i++) {
            int num = Integer.parseInt(arr[i]);
            max = Math.max(max, num);
            min = Math.min(min, num);
        }
        return min + " " + max;
    }
}

File: sanghoon/programmers-2/최댓값과 최솟값.py
def solution(s):
    s = s.split()
    for i in range(len(s)) :
        s[i] = int(s[i])
    # print(str(min(s)) + " " + str(max(s)))
    return str(min(s)) + " " + str(max(s))

s = "1 2 3 4"
solution(s)